src/test/java/jrm/user/UserSignUpTest.java

```diff
@@ -1,6 +1,7 @@
 package jrm.user;
 
 
+import groovy.transform.builder.Builder;
```

Comment:
Why do we need groovy part in the java code?

Reply:
I'm straggling to answer this question. 
Looks like we don't need it here at all.